staging/src/k8s.io/apiserver/pkg/server/options/flow_control.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	"k8s.io/apiserver/pkg/apis/apiserver"
+	utilflowcontrol "k8s.io/apiserver/pkg/util/flowcontrol"
+	"k8s.io/utils/path"
+)
+
+// FlowControlOptions holds the flow control options.
+type FlowControlOptions struct {
+	// ConfigFile is the file path with flow control configuration
+	ConfigFile string
```

Comment:
If I understand correctly this is optional; if so, I recommend documenting that here.

Reply:
This is modelled after https://github.com/kubernetes/kubernetes/blob/a10bdfed09e96f3c2703454caafa2c884914e064/staging/src/k8s.io/apiserver/pkg/server/options/egress_selector.go#L32-L33

I don't see any field being documented as `optional` in package `options`. Please confirm that you want this change to happen.